CRM/Report/Form/Contribute/Detail.php

```diff
@@ -239,6 +241,7 @@ public function __construct() {
           'contribution_status_id' => array('title' => ts('Contribution Status')),
           'payment_instrument_id' => array('title' => ts('Payment Method')),
           'receive_date' => array('title' => ts('Date Received')),
+          'receive_date' => array('title' => ts('Thank-you Date')),
```

Comment:
@MegaphoneJon Should this be 
      `    'thankyou_date' => array('title' => ts('Thank-you Date')),`

Reply:
good spotting @yashodha !